plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/CommitParams.java

```diff
@@ -68,11 +83,8 @@ public CommitParams withContainer(@NotNull String container) {
      * @param repository
      *         full repository name
      * @return this params instance
-     * @throws NullPointerException
-     *         if {@code repository} is null
      */
-    public CommitParams withRepository(@NotNull String repository) {
```

Comment:
maybe there is a missing notice explaining in the PR or commit why current status is wrong
it contains only the "how you fix it" but not "why it requires a fix"
Also I suppose some tests are missing